### ISSUE TITLE
Placement offset

### ISSUE THREAD

**Issue by Anfoone** [Beardedrasta/RHUD#3]
Would it be possible to add a placement offset? It would be a great feature so that we can apply it to one of the HUD anchors, while still having the ability to slide it on the X or Y axis. Useful for when I need to resize my window

**Comment by Beardedrasta** [comment]
I will look into this, As im updating because something updated and broke the plugin.

**Comment by Anfoone** [comment]
That would be amazing! Looking forward to seeing this once you flush it out :)

**Comment by Beardedrasta** [comment]
Just waiting on them to pull the update into runelite hub. Whenever that may be :P

**Comment by Beardedrasta** [comment]
Ill let you know when they pull it through then close out this chat. I hate the way runelite works, I already had my plugin uploaded but since i had to fix so many things that THEY broke now i have to wait weeks for them to "feel like" checking it to be reuploaded. Its silly.

**Comment by Anfoone** [comment]
Oh geez that sounds like a pain! I understand the checks and balances but its a lot... Thanks for the update on this and for taking the time to implement this  idea!

**Comment by Beardedrasta** [comment]
WE ARE LIVE BABY, RHUD is updated.

**Comment by Anfoone** [comment]
IT'S HERE! Thank so you much for this!

One small ask, 100 offset is close to what I need but not all the way there, any way to bump it up to 150 or 200 for the next update?

**Comment by Beardedrasta** [comment]
Absolutely, I wasn't sure how big your screen was or how much people would need and didnt want to overkill. Ill add that to the next update :)

**Comment by cba133713-sudo** [comment]
UI is stuck on the bottom left for me with the offset being to limited to get the bar to the top, free arrange or unlimited offset would fix that issue and prevent further problems with it so people can just put it where they want no matter whats runelites current patch
